base/docs/helpdb/Base.jl

```diff
@@ -914,9 +914,9 @@ generate an array of such random numbers.
 randexp
 
 """
-    chop(string)
+    chop(string, i = 1)
```

Comment:
I agree that `i` should be changed, but `len` sounds like it would be the desired length of the resulting string rather than the length of the section to remove. I think `n` would be the more natural choice.